src/components/ui/common/stadisticscard.tsx

```diff
@@ -0,0 +1,16 @@
+"use client"
+import CountUp from 'react-countup';
+import { StadisticsCardProps } from '@/lib/definitions';
+import phrases from '@/assets/locales/es.json';
+import MoreButton from '@/components/ui/common/MoreButton';
+
+const StadisticsCard = ({ value, text }: StadisticsCardProps) => {
```

Comment:
The component name "Stadistics" contains a spelling error. It should be "Statistics".

```suggestion
const StatisticsCard = ({ value, text }: StadisticsCardProps) => {
```